pkg/capabilities/capabilities.go

```diff
@@ -31,6 +31,10 @@ type Capabilities struct {
 
 	// PerConnectionBandwidthLimitBytesPerSec limits the throughput of each connection (currently only used for proxy, exec, attach)
 	PerConnectionBandwidthLimitBytesPerSec int64
+
+	// EnableSELinuxIntegration controls whether SELinux
```

Comment:
I think this comment should define the semantics that you can expect from this capability in detail.